docs/site/Decorators.md

```diff
@@ -72,59 +76,63 @@ controller. For example:
 })
 class MyController {
   // The operation endpoint defined here will be overriden!
-  @get('/foo')
-  @param.query.number('limit')
-  greet(name) {}
+  @get('/greet')
+  greet(@param.query.number('limit') name: string) {}
 }
 app.controller(MyController);
 ```
 
 A more detailed explanation can be found in
-[Specifying Controller APIs](controllers.htm#specifying-controller-apis)
+[Specifying Controller APIs](Controllers.md#specifying-controller-apis)
 
 ### Operation Decorator
 
 Syntax:
-[`@operation(verb: string, path: string, spec?: OperationObject)`](http://apidocs.loopback.io/@loopback%2fcore/#818)
+[`@operation(verb: string, path: string, spec?: OperationObject)`](https://github.com/strongloop/loopback-next/blob/d6a7d05122280f36d582365aeaa58e80fe31ad8f/packages/openapi-v3/src/operation-decorator.ts#L79)
```

Comment:
What are the benefits of linking to the source code as opposed to linking to the `apidocs`?

Reply:
I don't remember honestly lol. I'll switch it back to apidocs

Reply:
they're more visually appealing IMO 🖌